.template/spec/base/template_spec.rb

```diff
@@ -36,6 +36,10 @@
   end
 
   context 'OpenAPI docs' do
```

Comment:
Let's move these tests inside the OpenAPI add-on :)

Reply:
Fixed in c162277 ✅